sdk/storage/Azure.ResourceManager.Storage/api/Azure.ResourceManager.Storage.netstandard2.0.cs

```diff
@@ -82,6 +82,23 @@ public partial class BlobInventoryPolicyResource : Azure.ResourceManager.ArmReso
         public virtual Azure.Response<Azure.ResourceManager.Storage.BlobInventoryPolicyResource> Get(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Storage.BlobInventoryPolicyResource>> GetAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public partial class BlobRestoreStatusRestoreBlobRangesOperation : Azure.ResourceManager.ArmOperation<Azure.ResourceManager.Storage.Models.BlobRestoreStatus>
```

Comment:
Currently we use {ReturnType}{MethodName}Opeation to generate the class name, but it seems sometimes the generated name will be strange ==

Reply:
Now I changed the pattern to {Resource}{MethodName}Opeation